Makefile

```diff
@@ -1,2 +1,56 @@
+all: whatdoesitdo tests authors
+
 whatdoesitdo:
+	@# generate the README "What does it do?" section
 	@./gen-readme.sh
+
+# To decrease tests verbosity, comment out unneeded targets
+tests: downloadffprefs checknotcovered checkdeprecated stats cleanup
```

Comment:
Cool, that's what I thought. I'm still not sure as to where and how to add it.

e.g., should it be like:

```
.NOTPARALLEL: downloadffprefs
```

Or just as it's own line? It wasn't completely clear to me.